src/webauth/index.ts

```diff
@@ -40,13 +24,11 @@ const callbackUri = (domain: string, customScheme?: string) => {
  * @see https://auth0.com/docs/api-auth/grant/authorization-code-pkce
  */
 class WebAuth {
-  private client;
   private domain;
   private clientId;
   private agent;
```

Comment:
Since now it's all being handled by Auth0.Android and Auth0.swift, do we still need the `agent`?